engine/src/main/java/org/terasology/rendering/opengl/FrameBuffersManager.java

```diff
@@ -252,109 +344,123 @@ public void saveScreenshot() {
             }
         };
 
-        CoreRegistry.get(ThreadManager.class).submitTask("Write screenshot", task);
-
+        threadManager.submitTask("Write screenshot", task);
```

Comment:
I'd leave it as it is.

I think that the ThreadManager should be injected with `@In` in the future. In any case, I guess this code will never be executed if the thread manager creation failed before.